ports/stm32/sdcard.c

```diff
@@ -156,6 +167,15 @@ void HAL_SD_MspInit(SD_HandleTypeDef *hsd) {
     // enable SDIO clock
     SDMMC_CLK_ENABLE();
 
+    #if defined(STM32H7)
+    // Enable MDMA clock
+    __HAL_RCC_MDMA_CLK_ENABLE();
+
+    // Reset SDMMC
+    __HAL_RCC_SDMMC1_FORCE_RESET();
```

Comment:
Actually I didn't test without a reset, I just found it like that in the examples. Also not sure if MDMA clock has to be enabled, but it just makes sense to enable it (could be enabled in early board init ?).

Reply:
If the reset is in the examples then it would be worth keeping it here.

But if the MDMA is not needed then it shouldn't be enabled.  It'll consume power.  Can you test without the MDMA?